psc/src/main/java/com/pinterest/psc/common/event/PscEvent.java

```diff
@@ -0,0 +1,39 @@
+package com.pinterest.psc.common.event;
+
+import com.pinterest.psc.common.TopicUri;
+import com.pinterest.psc.common.TopicUriPartition;
+
+import java.util.Map;
+
+public class PscEvent {
```

Comment:
Can you please document this class? Since we're adding it to core PSC lib, what's its significance? Does it work for Kafka/MemQ the same way?

Reply:
Done